Optimize abstracts of ISS (CQA) (#4941)
theforeman/foreman-documentation: guides/common/modules/proc_importing-exports-in-the-syncable-format.adoc

= Importing exports in the syncable format

[role="_abstract"]

> Import syncable exports from an allowed import path into the Library environment when the downstream organization is configured for export sync.

Reviewer: ```suggestion
Import syncable exports from an allowed import path into the Library environment.
```

Alternative:

```suggestion
Import syncable exports from an allowed import path on your {ProjectServer} into the Library environment.
```

I would shorten this; because you export first and import afterwards, IMO it does not make sense to mention this. IMO line 10+12 is enough.

Author: ```suggestion
Import syncable exports from an allowed import path on your downstream {ProjectServer} into the Library environment.
```
